components/eamxx/cmake/machine-files/aurora.cmake

```diff
@@ -11,15 +11,6 @@ set(EKAT_MPI_THREAD_FLAG "-d" CACHE STRING "")
 SET(SYCL_COMPILE_FLAGS "-std=c++17 -fsycl -fsycl-device-code-split=per_kernel -fno-sycl-id-queries-fit-in-int -fsycl-unnamed-lambda")
 SET(SYCL_LINK_FLAGS "-fsycl -fsycl-link-huge-device-code -fsycl-device-code-split=per_kernel  -fsycl-targets=spir64_gen -Xsycl-target-backend \"-device 12.60.7\"")
 
-set(CMAKE_CXX_FLAGS " -\-intel -Xclang -fsycl-allow-virtual-functions -mlong-double-64 -O3 -DNDEBUG ${SYCL_COMPILE_FLAGS}" CACHE STRING "" FORCE)
-set(CMAKE_Fortran_FLAGS "-fc=ifx -O3 -DNDEBUG -DCPRINTEL -g" CACHE STRING "" FORCE)
-set(CMAKE_C_FLAGS "-O3 -DNDEBUG" CACHE STRING "" FORCE)
+set(CMAKE_CXX_FLAGS " -\-intel -Xclang -fsycl-allow-virtual-functions -mlong-double-64 ${SYCL_COMPILE_FLAGS}" CACHE STRING "" FORCE)
+set(CMAKE_Fortran_FLAGS "-fc=ifx -DCPRINTEL -g" CACHE STRING "" FORCE)
```

Comment:
If this file is getting included in E3SM (instead of just standalone unit-test) builds, then this could be cleaned up too:
- `-fc=ifx` is not needed, `ifx` is the default fortran compiler
- `-DCPRINTEL -g` is added by `cime_config/machines/cmake_macros/oneapi-ifxgpu.cmake`